### PR TITLE
Escaping changes/fixes

### PR DESCRIPTION
**Pull request by jeremydstone**
Per team discussion. Let's have any PR discussion on slack for this one.

### CODE REVIEW

File: dashboard/app/views/levels/_unplug.html.haml [hunk 1]
```diff
@@ -59,7 +59,7 @@
       fallbackResponse: '#{@fallback_response.to_json}',
       callback: '#{@callback}',
       app: 'unplug',
-      level: '#{@level.level_num}',
+      level: '#{escape_javascript(@level.level_num)}',
```
Comment (ewjordan) [inline dashboard/app/views/levels/_unplug.html.haml:62]: For my own understanding, is this primarily done to inhibit the warning, or is this usage of level actually more dangerous than the others, where there are comments about sourcing from static data?

**Comment by ashercodeorg** [comment]
In my world-view, we minimize the amount of assuming that DB data is secure. Else, it is easy to use levelbuilder access as a vector into the system.

File: dashboard/app/views/levels/_unplug.html.haml (hunk 1 above)
Comment (jeremydstone) [reply dashboard/app/views/levels/_unplug.html.haml:62]: Answered in Slack

Comment (breville) [reply dashboard/app/views/levels/_unplug.html.haml:62]: What was the answer? :)

**Pull request merged**